packages/react-dom-bindings/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -255,17 +267,79 @@ export function createResponseState(
       bootstrapChunks.push(endAsyncScript);
     }
   }
+
+  const fallbackBootstrapChunks = [];
+  if (fallbackBootstrapScriptContent !== undefined) {
+    fallbackBootstrapChunks.push(
+      inlineScriptWithNonce,
+      stringToChunk(
+        escapeBootstrapScriptContent(fallbackBootstrapScriptContent),
+      ),
+      endInlineScript,
+    );
+  }
+  // We intentionally omit the rizz runtime for fallback bootstrap even if configured.
+  // Even if it is configured the fallback bootstrap only executes if React errors at some Root
+  // Boundary and in these cases there will be no instructions for the runtime to execute
+  if (fallbackBootstrapScripts !== undefined) {
```

Comment:
Q: Could `writeInitialResources` called when the root suspense / container boundary fails? From my limited understanding it is, but definitely could be wrong.

Asking since `writeInitialResources` is currently the only function that sends the external runtime (i.e. the external runtime is not sent with `bootstrapChunks`)